core/include/cubos/core/ecs/dispatcher.hpp

```diff
@@ -9,81 +9,199 @@
 #include <string>
 #include <memory>
 
+#define ENSURE_CURR_SYSTEM()                                                                                           \
```

Comment:
Wrap these macros with `do {...} while(false)` to force using `;` after it.

Reply:
Fixed